### ISSUE TITLE
Reformat code deletes comments in pair macros

### ISSUE THREAD

**Issue by mrtnzlml** [JanTvrdik/intellij-latte#23]
Minimal code:
```latte
{if true}
	{* FIXME *}
{/if}
```
Or (not related to the `if`):
```latte
{block tada}
	{* FIXME *}
{/block}
```
Ater running "Reformat code" (Ctrl+Alt+L) the comment disappears. Also it's not possible to copy and paste comment in this block.

**Comment by matej21** [comment]
hi, this should be fixed by 8ff7eec, could you please try 0.3.2 version?

**Comment by mrtnzlml** [comment]
Seems good. Sorry, I didn't see this issue.
